examples/amp-story/interactive_results.html

```diff
@@ -0,0 +1,155 @@
+    <script async src="https://cdn.ampproject.org/v0.js"></script>
+    <script async custom-element="amp-story" src="https://cdn.ampproject.org/v0/amp-story-1.0.js"></script>
+    <style amp-custom>
+    body {
+      font-family: 'Poppins', sans-serif;
+    }
+
+    amp-story-interactive-quiz, amp-story-interactive-results-category {
+      align-self: center;
+      justify-self: center;
+    }
+    </style>
+  </head>
+  <body>
+    <amp-story
+        standalone
+        title="Interactive Story"
+        publisher="AMP Story testing"
+        publisher-logo-src="example.com/logo.png"
+        poster-portrait-src="example.com/poster.jpg">
+        <amp-story-page id="cover">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                animate-in="pan-left"
+                animate-in-duration="30s"
+                src="https://hips.hearstapps.com/hmg-prod.s3.amazonaws.com/images/dog-puppy-on-garden-royalty-free-image-1586966191.jpg"
+                layout="responsive"
+                height="1600"
+                width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="thirds">
+            <div style="color:white;padding:10px;align-self:end" grid-area="lower-third">
+              <h1>What adorable animal are you?</h1>
+              <p>The ultimate animal quiz</p>
+              <p>Provided with love by the AMP team</p>
+            </div>
+          </amp-story-grid-layer>
+        </amp-story-page>
+        <amp-story-page id="quiz-1">
+          <amp-story-grid-layer template="fill">
+            <amp-img
+                src="img/green-stuff.jpg" layout="responsive" height="1600" width="900">
+            </amp-img>
+          </amp-story-grid-layer>
+          <amp-story-grid-layer template="center">
+            <amp-story-interactive-quiz
+                prompt-text="Whats your favorite food?"
+                option-1-text="Bones" option-1-category="Dog" option-1-correct
+                option-2-text="Fish" option-2-category="Cat" option-2-correct
+                option-3-text="Carrots" option-3-category="Bunny" option-3-correct
+                option-4-text="Cheese" option-4-category="Mouse" option-4-correct>
+            </amp-story-interactive-quiz>
```

Comment:
I'm using quizzes because the muli-option polls are not created yet, but the important part is the option-#-category that is implemented on the abstract side, so it technically works on all interactives, and we can limit it based on the validation rules (or overriding updateStoreState_).